src/java.base/share/classes/java/net/doc-files/net-properties.html

```diff
@@ -267,13 +267,21 @@ <H2>Address Cache</H2>
 	policy, while a value of 0 (zero) means no caching. The default value
 	is -1 (forever) if a security manager is installed, and implementation-specific
 	when no security manager is installed.</P>
+	<LI><P><B>{@systemProperty networkaddress.cache.stale.ttl}</B> (default: see below)<BR>
+	Value is an integer corresponding to the number of seconds the stale names
+	will be kept in the cache. The name is considered stale if it expired and
+	attempts to refresh it were unsuccessful. This property is useful if it is
+	preferable to use a stale name rather than result of unsuccessful name lookup.
+	A value of 0 (zero) or if the property is not set means do not use stale
+	names, the negative value is ignored.
```

Comment:
I'd suggest to rephrase:

```suggestion
	A value of 0 (zero) or if the property is not set means do not use stale
	names. Negative values are ignored.
```